docs/release_notes.adoc

```diff
@@ -13,40 +15,73 @@ include::include.adoc[]
+* Improve StackTraceFilter by ignoring `java.lang.invoke.*` spockPull:1892[]
 * Improve `@TempDir` field injection, now it happens before field initialization, so it can be used by other field initializers.
+* Improve Spock-Compiler does not use wrapper types anymore spockPull:1765[]
+* Improve reduce lock contention of the `byte-buddy` mock maker, when multiple mocks are created concurrently spockPull:1778[]
+* Improve `@Use` on feature and fixture method for parallel execution spockPull:1691[]
+* Improve IDE support
+** by adding a closure signature hint that derives closure argument type from variable type spockPull:1785[]
+** by adding missing closure hints for Spy(T, Closure) spockPull:1786[]
+** by adding GDSL/DSLD for ConditionalExtension's annotations spockPull:1808[]
+** by making `spock.gdsl` fail-safe and cover some more cases spockPull:1783[]
+* Improve RunContexts behavior in multithreaded executions spockPull:1758[], spockPull:1846[]
+* Improve generic handling by replacing `gentyref` code with https://github.com/leangen/geantyref[geantyref] library spockPull:1743[]
+* Improve handling of passing `null` to `thrown()` to be consistent with mock object creation spockPull:1799[]
+* Improve `old(...)` calls by validating their argument count spockPull:1810[]
+* Improve robustness of the AST transformation against missing classes in the compile classpath spockPull:1704[]
+* Improve handling of `shared.` conditions in `@IgnoreIf`/`@Requires`, the condition is now checked before creating data providers spockPull:1711[]
+** This is now a required dependency used by spock: `io.leangen.geantyref:geantyref:1.3.14`
+* Fix issue with mocks of Groovy classes, where the Groovy MOP for `@Internal` methods was not honored by the `byte-buddy` mock maker spockPull:1729[]
+** This fixes multiple issues with Groovy MOP: spockIssue:1501[], spockIssue:1452[], spockIssue:1608[] and spockIssue:1145[]
+* Improve support for generic return types for mocks spockPull:1731[]
+** This fixes the issues: spockIssue:520[], spockIssue:1163[]
+* Fix the lifecycle of simple features spockPull:1675[]
 * Fix exception when configured `baseDir` was not existing, now `@TempDir` will create the baseDir directory if it is missing.
 * Fix bad error message for collection conditions, when one of the operands is `null`
+* Fix docs about initializer method interceptors spockPull:1666[]
 * Fix possible deadlock, when blocking in mock response generators spockPull:1885[]
+* Fix SpringSpy not working with `DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD` spockPull:1869[]
+* Fix null handling for collection conditions spockPull:1858[]
+* Fix interceptor contexts spockPull:1676[]
+* Fix properly call `IRunListener.specSkipped` and `.featureSkipped` spockPull:1811[]
+* Fix defining responses for additionalInterfaces spockPull:1730[]
+* Fix lost execution mode for iteration nodes of implied data-driven features spockPull:1615[]
```

Comment:
Line 77 `This fixes the issues`  should be moved to line 71 below `Fix possible deadlock,`

Reply:
good catch